packages-web/calendar/src/components/CalendarContainer.ts

```diff
@@ -81,7 +82,7 @@ class CalendarContainer extends Component<Container.CalendarContainerProps, Cale
     }
 
     componentWillUnmount(): void {
-        this.subscriptionHandles.forEach(window.mx.data.unsubscribe);
+        this.subscriptionContextHandles.forEach(window.mx.data.unsubscribe);
```

Comment:
```suggestion
        this.subscriptionContextHandles.forEach(window.mx.data.unsubscribe);
        this.subscriptionEventHandles.forEach(window.mx.data.unsubscribe);
```

Reply:
Added the unsubscribe for all calendar events